routers/web/web.go

```diff
@@ -1484,11 +1484,11 @@ func registerRoutes(m *web.Router) {
 			})
 			m.Group("/code-frequency", func() {
 				m.Get("", repo.CodeFrequency)
-				m.Get("/data", repo.CodeFrequencyData)
+				m.Get("/data", repo.ContributorStats)
```

Comment:
![image](https://github.com/user-attachments/assets/4ebdb718-c33a-4bf1-b0ba-1d4f2b2e5fd2)

Reply:
The functions are duplicated.

Reply:
What do you mean by "duplicated"?

I mean the old names are right.

Reply:
The function body are the same between `repo.CodeFrequencyData` and `repo.RecentCommitsData`

Reply:
OK, I can see it is duplicated.

But the new function name and comment don't seem accurate. Old name and comment are better.